config/options.go

```diff
@@ -651,6 +657,16 @@ func (o *Options) Validate() error {
 		return fmt.Errorf("config: server must be run with `autocert`, " +
 			"`insecure_server` or manually provided certificates to start")
 	}
+
+	switch o.Provider {
+	case azure.Name, github.Name, gitlab.Name, google.Name, okta.Name, onelogin.Name:
+		if len(o.Scopes) > 0 {
+			docLink := "https://www.pomerium.io/reference/#identity-provider-scopes"
```

Comment:
could you move whole error string to const at top of the tin?

Reply:
Done